cli2/lib/morphir-elm.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
Same here. No need to add it.

Reply:
File is still in the PR. It should be removed from it. You can put them in .gitignore